src/fw/test/integration/in-zone-enabled.test.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2014, Joyent, Inc. All rights reserved.
+ * Copyright (c) 2019, Joyent, Inc. All rights reserved.
```

Comment:
Are we gonna keep `(c) 2019,`? I thought we've moved towards simply `2019` for these notices.

Reply:
Good spotting - these should use the simpler version.